R/plot_with_settings.R

```diff
@@ -191,7 +192,8 @@ plot_with_settings_ui <- function(id) {
 #'   ),
 #'   server = function(input, output, session) {
 #'     plot_r <- reactive({
-#'       ggplot2::qplot(x = 1:5, y = 1:5)
+#'       ggplot2::ggplot(NULL) +
```

Comment:
ggplot2::ggplot()  more conventional and concise .

Reply:
Agreed, Do you have any changes to this that you'd like?